src/test/java/ICS_Account_functionality.java

```diff
@@ -0,0 +1,115 @@
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.junit.Assert;
+import org.openqa.selenium.*;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.chrome.ChromeOptions;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class ICS_Account_functionality {
+    public static void main(String[] args) throws InterruptedException {
+        accountTest("desktop");
+        accountTest("tablet");
+        accountTest("mobile");
+    }
+
+    public static void accountTest(String size) throws InterruptedException {
+        WebDriverManager.chromedriver().setup();
+
+        WebDriver driver = driverSetup(size);
+
+        Dimension desktop = new Dimension(1366, 766);
+        Dimension tablet = new Dimension(766, 1024);
+        Dimension mobile = new Dimension(400, 800);
+
+        if (size.equals("desktop")) {
+            driver.manage().window().setSize(desktop);
+        } else if (size.equals("tablet")) {
+            driver.manage().window().setSize(tablet);
+        } else {
+            driver.manage().window().setSize(mobile);
+        }
+
+        driver.get("https://icecreamshop-frontend-rkbemzbp2q-ez.a.run.app/");
+
+        //check if "Account" button from Homepage it's written correctly
+        if (size.equals("desktop") || size.equals("tablet")) {
+            String accountButton = driver.findElement(By.cssSelector("[class='TopSection_accountLabel__1nFim']")).getText();
+            boolean condition1 = false;
+            if (accountButton.equals("Account")) {
+                condition1 = true;
+            }
+            Assert.assertTrue(condition1);
```

Comment:
An easier way to assert would be like this : 
Assert.assertEquals(condition1, "Account"); - thus having only 1 line of code instead of 5